.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v3
         with:
-          node-version: 12.x
```

Comment:
I'd revert these if node 12 isn't being removed

Reply:
So the problem is that `@ember/test-helpers@^2.7.0` has a dependency on `@embroider/util@^1.9.0` that resolves to `@embroider/util@1.11.0`.

```sh
error @embroider/util@1.11.0: The engine "node" is incompatible with this module. Expected version "14.* || >= 16". Got "12.22.12"
```

What should we do now?

Reply:
I think it's fine to drop node 12, support has ended for it a while back (on the node front).

Reply:
Alright. Droped it and added the breaking label again.